demo-output.yaml

```diff
@@ -212,29 +154,6 @@
           file: file:///analyzer-lsp/examples/golang/main.go
           lineNumber: 9
       extras: []
-    golang-gomod-dependencies:
```

Comment:
Note that this is not there, because the dependency service is not added for external providers. 

I figure this could be a follow up

Reply:
was there additional difficulty moving that interface over, or did you just want to keep the PR smaller?

Reply:
I was trying to keep it small but might as well do this as well.